config/migrations/2024/contact/20240116200259-add-contact-session-role-to-all-types-of-organizations.sparql

```diff
@@ -0,0 +1,25 @@
+PREFIX ext:     <http://mu.semte.ch/vocabularies/ext/>
+PREFIX skos:    <http://www.w3.org/2004/02/skos/core#>
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX foaf:    <http://xmlns.com/foaf/0.1/>
+
+#
+# Insert contact app session role for all types of
+# organizations. 
+#
+
+INSERT {
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    ?account ext:sessionRole "abb_organisatieportaal_rol_3d" .
+  }
+}
+WHERE {
+  ?bestuurseenheid a besluit:Bestuurseenheid .
+
+  ?persoon a foaf:Person ;
+    foaf:member ?bestuurseenheid ;
+    foaf:account ?account .
+
+  ?account a foaf:OnlineAccount ;
+    ext:sessionRole ?sessionRole .
```

Comment:
I found some migrations that also check the `foaf:accountServiceHomepage` value:
https://github.com/lblod/app-digitaal-loket/blob/82a4afa7e5aaacf2cd0303807cb610c460b2c8e6/config/migrations/2022/20220809162724-ipdc-lpdc/20220520131500-add-role-to-mock-users.sparql#L16

That's not needed here? (Not an expert 😅)

Reply:
`ext:sessionRole ?sessionRole .` is this part needed, doesn't seem to be used?

Reply:
> I found some migrations that also check the `foaf:accountServiceHomepage` value:
> 
> https://github.com/lblod/app-digitaal-loket/blob/82a4afa7e5aaacf2cd0303807cb610c460b2c8e6/config/migrations/2022/20220809162724-ipdc-lpdc/20220520131500-add-role-to-mock-users.sparql#L16
> 
> That's not needed here? (Not an expert 😅)

I guess it's needed since querying the database leads to two types of `accountServiceHomepage`; we need to only apply the session role for the `mock-login-service` accounts.

Reply:
> `ext:sessionRole ?sessionRole .` is this part needed, doesn't seem to be used?

You're right; good catch.